### PR TITLE
Removed the `api` command from the Circleci workflow

### LINKED ISSUE #1303: 🐈 Task: Update Circle CI test script to remove deprecated api.
### Summary

With the removal of the `api` command in ersilia, any workflows that make use of it would naturally fail, one such example being the Circle CI [workflow](https://github.com/ersilia-os/ersilia/blob/master/.circleci/config.yml). We simply need to remove the `api` command here. 

### Objective(s)

_No response_

### Documentation

_No response_

### PR DESCRIPTION
**Pull request by mercybassey**
Thank you for taking your time to contribute to Ersilia, just a few checks before we proceed
- [x] Have you followed the guidelines in our [Contribution Guide](https://github.com/ersilia-os/ersilia/blob/master/CONTRIBUTING.md)
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

**Description**

The `api` command in Ersilia is deprecated; therefore, any workflow that uses it will fail. 

**Changes to be made**

This PR aims to remove the  `api` command from the Circle CI workflow.

**Status**

This PR removes the `api` command from the Circle CI workflow.

Is this pull request related to any open issue? If yes, replace issueID below with the issue ID

Closes #1303

### CODE REVIEW

**Comment by mercybassey** [comment]
Hello @DhanshreeA this PR is ready for review. Thanks.

File: .circleci/config.yml
Comment (DhanshreeA) [inline .circleci/config.yml:1]: Hi @mercybassey we actually need to use the `run` command, instead of the `api` command.

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

File: .circleci/config.yml
Comment (mercybassey) [reply .circleci/config.yml:1]: Okay. I'll fix it

**Comment by mercybassey** [comment]
@DhanshreeA I have added the `run` command instead. Thanks.

**Review by DhanshreeA — APPROVED** [review]

**Pull request merged**